crates/types/src/cluster/cluster_state.rs

```diff
@@ -121,24 +121,45 @@ pub struct SuspectNode {
     pub last_attempt: MillisSinceEpoch,
 }
 
-#[derive(Debug, Clone, Copy, Serialize, Deserialize, Eq, PartialEq, IntoProst, strum::Display)]
+#[derive(
+    Debug,
+    Clone,
+    Copy,
+    Serialize,
+    Deserialize,
+    Eq,
+    PartialEq,
+    IntoProst,
+    strum::Display,
+    bilrost::Enumeration,
+)]
 #[prost(target = "crate::protobuf::cluster::RunMode")]
 #[strum(serialize_all = "snake_case")]
 pub enum RunMode {
-    Leader,
-    Follower,
+    Follower = 0,
```

Comment:
Bilrost requires numeric enums to have an "EmptyState" which is always the variant with zero value. It's up to us to introduce `Unknown` variant (like what we do with protobuf) or assign the 0 value to one of the existing variants.